src/windows/views/properties_tableview.py

```diff
@@ -152,8 +152,18 @@ class PropertiesTableView(QTableView):
     def mouseMoveEvent(self, event):
         # Get data model and selection
         model = self.clip_properties_model.model
-        row = self.indexAt(event.pos()).row()
-        column = self.indexAt(event.pos()).column()
+
+        # Do not change selected row during mouse move
+        if self.lock_selection:
+            row = self.prev_row
+        else:
+            row = self.indexAt(event.pos()).row()
+            self.prev_row = row
+            self.lock_selection = True
+
+        if row is None:
```

Comment:
Looks like `row` could be undefined at this point. Probably still need a declaration outside these `if/else` statements

Reply:
I googled a bit and it seems that the code like this is common in Python.

The `None` check (the later one) was left from the debugging and not a big deal right now. Feel free to remove it. I placed it just as reminder for myself when code wasn't complete yet.

By the way, the `self.currentIndex().row()` instead of the `indexAt` - works OK too with this QTableView.